src/js/integrations/nativelinkederrors.ts

```diff
@@ -0,0 +1,231 @@
+import { exceptionFromError } from '@sentry/browser';
+import type {
+  DebugImage,
+  Event,
+  EventHint,
+  EventProcessor,
+  Exception,
+  ExtendedError,
+  Hub,
+  Integration,
+  StackFrame,
+  StackParser,
+} from '@sentry/types';
+import { isInstanceOf, isPlainObject } from '@sentry/utils';
+
+import type { NativeStackFrames } from '../NativeRNSentry';
+import { NATIVE } from '../wrapper';
+
+const DEFAULT_KEY = 'cause';
+const DEFAULT_LIMIT = 5;
+
+interface LinkedErrorsOptions {
+  key: string;
+  limit: number;
+}
+
+/**
+ * Processes JS and RN native linked errors.
+ */
+export class NativeLinkedErrors implements Integration {
```

Comment:
Does the integration handle well if its using an older version of RN that don't contain the linked errors at all?

Reply:
Yes, prev RN versions would not throw an error in JS at all or it would be a plain JS error.

Tested here: https://github.com/getsentry/sentry-react-native/pull/3201/files/d3ea823bbc2048300e907d76105991ea02e6c520#diff-efd6c71dcc055b46a67752fd77362be1c8bc3499caf7616416841302da5e8145R19-R74